app/helpers/puppetca_helper.rb

```diff
@@ -1,8 +1,22 @@
 module PuppetcaHelper
+  STATES = [N_('pending'), N_('valid'), N_('revoked')]
+  CA_ICONS = { 'valid' => 'pficon pficon-ok',
+               'pending' => 'pficon pficon-warning-triangle-o',
+               'revoked' => 'pficon pficon-error-circle-o' }
```

Comment:
I'm not sure revoked should be shown as error, its mostly FYI (as revoked is normal operation).